server.js

```diff
@@ -35,6 +35,31 @@ app.post('/items', function(req, res) {
   });
 });
 
+app.put('/items/:id', function(req, res) {
+  let id = req.params.id;
+  let newName = req.body.name;
+
+  if (id === 'undefined') res.status(400).send('Bad request, invalid ID');
+  if (newName === 'undefined' || newName === null) res.status(400).send('Bad request, no name given'); //This isn't working yet
```

Comment:
My mentor introduced me to the idea of "clean commits". What this means is if you have code that throws errors, breaks other code, doesn't belong, etc., to fix/remove it before committing. That way you can roll back to the commit in the future, without unintentionally breaking things. 

So if this line does any of that, you may want to add a TODO and just comment it out -- my suggestion at least.